aws/table_aws_elasticache_serverless_cache.go

```diff
@@ -0,0 +1,292 @@
+package aws
+
+import (
+	"context"
+	"errors"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/elasticache"
+	"github.com/aws/aws-sdk-go-v2/service/elasticache/types"
+
+	"github.com/aws/smithy-go"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsElastiCacheServerlessCache(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_elasticache_serverless_cache",
+		Description: "AWS ElastiCache Serverless Cache",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("serverless_cache_name"),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"ServerlessCacheNotFoundFault", "InvalidParameterValue"}),
```

Comment:
Do we still need to explicitly add the `InvalidParameterValue` error code to the ignore configuration here? Since this error code is already defined at the [plugin level](https://github.com/turbot/steampipe-plugin-aws/blob/main/aws/plugin.go#L23-L37), wouldn’t that handle it for us?